presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/StandardColumnMappings.java

```diff
@@ -288,16 +287,16 @@ public static ColumnMapping timeColumnMappingUsingSqlTime(ConnectorSession sessi
                 TIME,
                 (resultSet, columnIndex) -> {
                     Time time = resultSet.getTime(columnIndex);
-                    return NANOSECONDS.toMillis(toLocalTime(time).toNanoOfDay());
+                    return NANOSECONDS.toMillis(fromSqlTime(time).toNanoOfDay());
                 },
                 timeWriteFunctionUsingSqlTime(session));
     }
 
-    private static LocalTime toLocalTime(Time sqlTime)
+    private static LocalTime fromSqlTime(Time sqlTime)
     {
         // Time.toLocalTime() does not preserve second fraction
         return sqlTime.toLocalTime()
-                // TODO is the conversion correct if sqlTime.getTime() < 0?
+                // TODO there is an error if sqlTime.getTime() < 0
```

Comment:
This is not a style change, it's semantics. Seprate commit is warranted.
Also, do  you _know_ that?
(I put a question mark, because I _suspected_ but didn't _know_)

Reply:
I tested one. The documentation also says `withNano` only accepts arguments from 0 to 999 999 999.